smart-contracts/contracts/Unlock.sol

```diff
@@ -80,6 +84,7 @@ contract Unlock is
   {
     // We must manually initialize Ownable.sol
     Ownable.initialize(_owner);
+    globalBaseTokenURI = 'https://locksmith.unlock-protocol.com/api/key/';
```

Comment:
We can't initialize here, it won't be called again.  How about no default in the contract, and then we use the changeBase function to set it?

Reply:
of course! ha, this only works in testing. yep, will remove this and we'll have to set it explicitly with the setter func.

Reply:
done